controllers/auth.js

```diff
@@ -220,23 +218,66 @@ const uploadFile = async (req, res) => {
         [user_id, doc_id, key]
       );
     }
-    // this is to check the total number of document in a type preonboarding on on-borading
-    let no_total_docs_uplaoed=await db.query("SELECT COUNT(*) FROM uploaded_docs ud WHERE user_id=$1 AND all_docs_id in (Select id from all_docs where doc_type_id=$2) ",[user_id,doc_type_id]);
-    let total_doc=await db.query("SELECT dt.total FROM doc_type dt WHERE dt.id=$1 ",[doc_type_id]);
-    
-    if((no_total_docs_uplaoed.rows[0].count==total_doc.rows[0].total) &&(doc_type_id===1))//for pre-on borading
-    {
-      let uploaded_pre_onboarding_status =await db.query("UPDATE users SET uploaded_pre_on_board_docs= $1 WHERE id=$2",[true,user_id]);
-     logger.debug(`sussesfull updated the pre-onboarding for user ${user_id}`);
+    let adminEmail = [];
+    let admin = await db.query("select email from users where is_admin=true");
+    //storing email in a array for sending email
+    for (let i = 0; i < admin.rowCount; i++) {
+      adminEmail.push(admin.rows[i].email);
     }
-    if((no_total_docs_uplaoed.rows[0].count==total_doc.rows[0].total) &&(doc_type_id===2))//for onboarding
-    {
-      let uploaded_pre_onboarding_status =await db.query("UPDATE users SET uploaded_on_board_docs=$1 WHERE id=$2",[true ,user_id]);
-      logger.debug(`sussesfull updated the onboarding for user ${user_id}`)
-      
+    //check if user has uploaded all the documents - pre on boarding
+    let checkDocs1 = await db.query(
```

Comment:
There is a lot of code duplication in this function which can be reduced, please take a look at this